aopy/analysis/behavior.py

```diff
@@ -146,6 +147,61 @@ def compute_path_length_per_trajectory(trajectory):
     path_length = np.sum(lengths)
     return path_length
 
+def compute_movement_stats(trajectory, target_position, rotation_axis=np.array([1, 0]), error_axis=1, 
+                           return_all_stats=False):
+    """
+    Computes movement statistics of a trajectory relative to a target position. 
+
+    Args:
+        trajectory (nt, 2): The trajectory coordinates for each point in time. 
+        target_position (2,): Target position coordinates.
+        rotation_axis ((2,) array, optional): The axis onto which movement will be
+            projected to calculate error. Defaults to np.array([1, 0]).
+        error_axis (int, optional): axis (after rotation) along which to compute the error 
+            statistics. Default 1.
+
+    Returns:
+        tuple: A tuple containing:
+            |**mean (float):** The mean error of the trajectory relative to the target position.
+            |**std (float):** The variance of the error of the trajectory relative to the target position.
+            |**auc (float):** The area under the curve ofthe trajectory relative to the target position.
+        additionally, with return_all_stats=True: 
+            |**abs_mean (float):** The mean of the absolute value of the trajectory error.
+            |**abs_min (float):** The minimum absolute trajectory error.
+            |**abs_max (float):** The maximum absolute trajectory error.
+            |**abs_auc (float):** The area under the curve of the absolute value of the trajectory error.
+            |**sign (float):** 1 if the maximum positive value is bigger than the maximum negative value. -1 otherwise.
+            |**signed_min (float):** The minimum value if the sign is 1, otherwise the maximum value of the trajectory error.
+            |**signed_max (float):** The maximum value if the sign is 1, otherwise the minimum value of the trajectory error.
+            |**signed_abs_mean (float):** The sign multiplied by the absolute value of the mean trajectory error.
+        
```

Comment:
I think we should also return 'dist_ts'. I could see a scenario where someone wants the movement error at each timepoint of a trajectory.